src/domUpdates.js

```diff
@@ -61,7 +61,48 @@ let domUpdates = {
         <p>Room Type: ${sortedBooking.roomType}</p>
         <p>Bed Size: ${sortedBooking.bedSize}</p>
         <p>Beds: ${sortedBooking.numBeds}</p>
-        <p>Cost Per Night: ${sortedBooking.cost}</p>
+        <p>Cost Per Night: $${sortedBooking.cost}</p>
+      </div>
+      `
+    })
+  },
+
+  displayFutureBookings(customer, roomsData) {
```

Comment:
because displayFutureBookings and displayPast Bookings are almost identical functions, I can likely merge into one function and pass in an argument of the section I want to update.